config/routes.rb

```diff
@@ -7,11 +7,11 @@
       sessions: 'publishing_users/sessions'
     }
 
-  root to: 'tasks#index'
+  root to: 'tasks#my'
```

Comment:
Maybe we should route tasks to /tasks, we can redirect there from home if user is logged in.